rimage/depth_map_raw.go

```diff
@@ -83,26 +85,34 @@ func readDepthMapRaw(f *bufio.Reader, firstBytes int64) (*DepthMap, error) {
 	return setRawDepthMapValues(f, &dm)
 }
 
-func readDepthMapViam(f *bufio.Reader) (*DepthMap, error) {
-	dm := DepthMap{}
+func readDepthMapViam(ff io.Reader) (*DepthMap, error) {
+	f := bufio.NewReader(ff)
+	dm := &DepthMap{}
 
 	rawWidth, err := _readNext(f)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "could not read vnd.viam.dep width")
 	}
 	dm.width = int(rawWidth)
 	rawHeight, err := _readNext(f)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "could not read vnd.viam.dep height")
 	}
 	dm.height = int(rawHeight)
-
-	return setRawDepthMapValues(f, &dm)
+	// dump the rest of the bytes in a depth slice
+	datSlice := make([]Depth, dm.height*dm.width)
```

Comment:
This seems like a smoother way to do setRawDepthMapValues() except now there's two places with basically the same code chunk.  We only use this here and in readDepthMapRaw, so maybe just go ahead and replace the function with this logic? Might mean that the error printout gets slightly less specific though...

Reply:
Seems like we're just dumping it all out as opposed to reading 8 bytes at a time. Nice. This should also be valid for raw depth maps.

Reply:
Raw Depth maps are actually treated differently:

In the raw depth map function, it is writing the two bytes (uint16) of depth info from the `*DepthMap` to 8 bytes (uint64) of space in the buffer. These leaves a lot of 0-padding in the files. When reading the Raw Depth file using `setRawDepthMapValues`, it has to read 8 byte chunks at a time, just to read the 2 bytes of depth to put into the `*DepthMap` pixel.

This new Viam function, on the other hand, writes out 2 bytes, and reads 2 bytes. So the space the file takes up is smaller, because it doesn't have 6 extra bytes of 0s for every 2 bytes of depth info. This also allows you to just dump the data from the file into a depth slice, without having to parse 8 bytes at a time.

Reply:
I'll put a comment in the file pointing out the difference